### DISCUSSION TITLE
Can I provide data to a wasm file build?

### DISCUSSION THREAD

**Discussion by shkup** [open-policy-agent/community#569] [OPA and Rego]
When I create a regular bundle file I can provide both `rego` and data `json`.
Can I do the same with `wasm` build command?
How can I add a data file to this:
`opa build -t wasm -e example/allow example.rego`

**Comment by srenatus** [comment]
That's not possible right now. But the idea makes sense, perhaps some SDK could implement that.

**Reply by srenatus** [reply]
I misread your question, I thought you'd wanted the SDK to ingest a tarball with wasm+data and "do the right thing", but you want to build a tarball with that.
